apps/dashboard/app/views/batch_connect/shared/_breadcrumb.html.erb

```diff
@@ -0,0 +1,9 @@
+<nav aria-label="Breadcrumb">
+  <ol class="breadcrumb">
+    <% links.each do |link, idx| %>
+      <%= tag.li class: current_page?(link) ? 'breadcrumb-item' : 'breadcrumb-item active' do %>
+        <%= tag.a link[:text], href: (link[:href] unless link.key?('href') || link[:disabled]) %>
+      <% end %>
+    <% end %>
+  </ol>
+</nav>
```

Comment:
We talked off-line, but I think I figured out how to make this more accessible. 

We set `$breadcrumb-divider: '';` 

And include this scss (or similar) that'll shape the `::before` element
```css
.breadcrumb-item + .breadcrumb-item::before {
  display: inline-block;
  margin: 0.15em 1em 0 0;
  transform: rotate(15deg);
  border-right: 0.1em solid currentColor;
  padding-right: 0.5em;
  height: 75%;
}
```

Also for the last item, why not also have a link for it?  I don't really see the harm in case someone wants to refresh the page. We'd we need `aria-current="page"` to the last item (and maybe some additional css).

Reply:
slashes are read as text and it throws off the screen readers. It isn't bootstrap, though bootstraps examples aren't right.

So if you navigate through the breadcrumb list and it has 3 items (home / my interactive sessions / pitzer desktop)
1. The first item is read correctly
2. The second item reads: "slash my interactive sessions" (it doesn't read it as a link, only text)

When it's correct
1. The first item is read correctly
2. The second item reads: "my interactive sessions visited link"

I think the root of the issue is that `::before` is outside of the `a` element, so the screen reader doesn't recognize it as 1 logical thing. So the fix is to get rid of the text so that `::before` are purely visual slashes and not read at all.

Reply:
Note that this issue exists in the mainline today.  I just happen to catch it during this review.